src/Illuminate/Database/Schema/Grammars/SqlServerGrammar.php

```diff
@@ -35,7 +35,12 @@ class SqlServerGrammar extends Grammar
      */
     public function compileTableExists()
     {
-        return "select * from sysobjects where type = 'U' and name = ?";
+        return "select obj.* from sys.objects  as obj
+                join sys.schemas as schem
+                    on obj.schema_id = schem.schema_id
+                where obj.type = 'U'
+                and obj.name = ?
+                and schem.name = "; // schema concatenated by builder.
```

Comment:
What does this comment mean? Where does this happen?

Reply:
In the processor. found not better way todo this since if default schema should be used parameter is not possble (needs to user sql method)

Reply:
if no schema is provided SCHEMA_NAME() constant/sql server method is used, this cannot be parameterized.

would be nice if this compile method could just return a string + array of bindings but that is not how the interface is setup.

Reply:
Where in the processor? What file and line number?